documentation/configuration.rst

```diff
@@ -264,9 +264,10 @@ FLEXMEASURES_MAX_PLANNING_HORIZON
 
 The maximum horizon for making schedules.
 API users are not able to request longer schedules.
-Set to ``None`` to forgo this limitation.
+Can be set to a specific ``datetime.timedelta`` or to an integer number of planning steps, where the duration of a planning step is equal to the resolution of the applicable power sensor.
+Set to ``None`` to forgo this limitation altoghether.
 
-Default: ``timedelta(days=7, hours=1)``
+Default: ``2520`` (e.g. 7 days for a 4-minute resolution sensor, 105 days for a 1-hour resolution sensor)
```

Comment:
> I was just thinking about this, and it seems to me that this is a rather large default horizon. This will eat a lot of CPU budget.

This is not the default horizon (which is currently set to 2 days), but the default max horizon. I agree that this is a rather large value, but it is what the FlexMeasures scheduler can currently handle without the scheduling jobs timing out after 180 seconds.

I also thought of coupling this PR with the introduction of a config setting to allow FlexMeasures hosts to overwrite python-rq's default job timeout of 180 seconds for the scheduling queue, but I decided to save that for a separate PR. That would be a simple change in `app.py`, something like:
```
Queue(name="scheduling", default_timeout=app.config["FLEXMEASURES_PLANNING_TIMEOUT"])
```

Reply:
Oh yes, sorry for the oversight. Actually, I would advise we give `FLEXMEASURES_PLANNING_HORIZON` the same treatment (that it can be an int).

Reply:
I thought about that, too, but I didn't find the use case particularly convincing. Is it a worthwhile extra feature, or just resolves an itchy asymmetry?

Reply:
The latter. But these things matter for user experience. I would expect these two to work the same.
And I believe we are not breaking existing settings here. Now is the cheapest time to do this.

Reply:
> Now is the cheapest time to do this.

I started implementing your request, but it turns out this is probably not true. The `FLEXMEASURES_PLANNING_HORIZON` is used in API versions 1.2 and 1.3, which allow retrieving schedules for multiple sensors in a single API call. That code assumes that the planning horizon is the same for all of these sensors, which with an integer `FLEXMEASURES_PLANNING_HORIZON` would no longer be the case. So this would currently require refactoring of already deprecated code.

I'd say it is cheaper to implement your request after these API versions are sunset.

Reply:
Sounds reasonable, can we make a ticket for version 0.14?

Reply:
https://github.com/FlexMeasures/flexmeasures/issues/586